src/TemplateLibrary/Buttons/index.js

```diff
@@ -12,7 +12,7 @@ const UploadButton = styled.button`
   text-align: right;
   text-decoration: none;
   font-size: 0.87em;
-  color: #76777D;
+  color: #86878D;
```

Comment:
What is the background color at this point? I'm imagining this change is fine.

Reply:
👍

Reply:
Background color is #141F3C. You can view new color contrast using this link https://webaim.org/resources/contrastchecker/?fcolor=86878D&bcolor=141F3C